phpunit/class-rest-widgets-controller-test.php

```diff
@@ -132,6 +180,7 @@ private function setup_sidebar( $id, $attrs = array(), $widgets = array() ) {
 			$attrs
 		);
 
+		// TODO: Is this needed?
```

Comment:
Doesn't seem to replicate any core behavior. Maybe it's a safeguard to have some content in sidebars when testing, in case we can catch some bugs?

Reply:
I can't remember why I added this! Removing it doesn't cause tests to fail, though, so I've removed it in e06c903a3e0dbb6636faad14cebcb4c2c058c93f.